src/main/java/de/tum/in/www1/artemis/web/rest/dto/metrics/ExerciseStudentMetricsDTO.java

```diff
@@ -9,13 +9,15 @@
  * DTO for exercise student metrics.
  *
  * @param exerciseInformation     the information about the exercises
+ * @param categories              the categories of the exercises
  * @param averageScore            the average score of the students in the exercises
  * @param score                   the score of the student in the exercises
  * @param averageLatestSubmission the average relative time of the latest submissions in the exercises
  * @param latestSubmission        the relative time of the latest submission of the students in the exercises
  * @param completed               the ids of the completed exercises
+ * @param teamId                  the id of the team the student is in
  */
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public record ExerciseStudentMetricsDTO(Map<Long, ExerciseInformationDTO> exerciseInformation, Map<Long, Double> averageScore, Map<Long, Double> score,
-        Map<Long, Double> averageLatestSubmission, Map<Long, Double> latestSubmission, Set<Long> completed) {
+public record ExerciseStudentMetricsDTO(Map<Long, ExerciseInformationDTO> exerciseInformation, Map<Long, Set<String>> categories, Map<Long, Double> averageScore,
```

Comment:
Categories look like this:
![image](https://github.com/ls1intum/Artemis/assets/5898705/c76c4614-6256-4ad3-ace5-687846a0d521)

I can work with it like this but it would be easier for me to just get it as json object instead of an escaped json string.

Reply:
That's how they are stored, but I can certainly look into it 👍